fcrepo-audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -57,6 +57,7 @@
     static final String PROV = "http://www.w3.org/ns/prov#";
     static final String XSD = "http://www.w3.org/2001/XMLSchema#";
     static final String EVENT_TYPE = "http://id.loc.gov/vocabulary/preservation/eventType/";
+    static final String EVENT_NAMESPACE = "http://fedora.info/definitions/v4/event#";
```

Comment:
Is there a reason the `EVENT_NAMESPACE` is not being defined with the other namespaces?
https://github.com/fcrepo4-exts/fcrepo-camel/blob/master/src/main/java/org/fcrepo/camel/RdfNamespaces.java